WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift

```diff
@@ -519,7 +519,8 @@ extension SiteStatsInsightsTableViewController: SiteStatsInsightsDelegate {
 
             self.navigationController?.popToRootViewController(animated: false)
             RootViewControllerCoordinator.sharedPresenter.showReaderTab()
-            if let nc = WPTabBarController.sharedInstance().selectedViewController as? UINavigationController,
+            if let tabBarController = RootViewControllerCoordinator.sharedPresenter.rootViewController as? WPTabBarController,
```

Comment:
Same comment as mentioned above with accessing `WPTabBarController?` directly

Reply:
Done in e619798f281ec0481b75fe5d1801c02d71f72018 👍